netci.groovy

```diff
@@ -1897,7 +1897,7 @@ def static calculateBuildCommands(def newJob, def scenario, def branch, def isPR
 
                             if (enableCorefxTesting) {
                                 def fxRoot = "%WORKSPACE%\\_\\fx"
```

Comment:
$fxRoot is used later in the groovy, so this change is creating an implicit dependency between the two scripts that would best be avoided.  Why not just fix the horked definition of fxRoot right here and below?